src/marlin_stubs/G26.cpp

```diff
@@ -1,168 +1,158 @@
+#include <algorithm>
+
 #include "../../lib/Marlin/Marlin/src/inc/MarlinConfig.h"
 #include "../../lib/Marlin/Marlin/src/feature/host_actions.h"
 #include "../../lib/Marlin/Marlin/src/feature/safety_timer.h"
 #include "../../lib/Marlin/Marlin/src/gcode/gcode.h"
 #include "../../lib/Marlin/Marlin/src/module/motion.h"
 #include "../../lib/Marlin/Marlin/src/Marlin.h"
 #include "marlin_server.hpp"
-#include "marlin_client.hpp"
+//#include "marlin_client.hpp"
```

Comment:
remove this commented include?
You cannot use marlin_client in marlin thread anyway